main.sh

```diff
@@ -44,9 +40,26 @@ if [ -n "${GITHUB_OUTPUT+x}" ]; then
   echo "project=$ProjectName" >>"$GITHUB_OUTPUT"
   echo "axe_url=$SERVER_URL$AxeURL" >>"$GITHUB_OUTPUT"
   echo "issue_count=$IssueCount" >>"$GITHUB_OUTPUT"
+  echo "issues_over_a11y_threshold=$IssuesOverA11yThreshold" >>"$GITHUB_OUTPUT"
+fi
+
+echo "axe DevHub found $IssueCount accessibility violations."
+
+if [ "$ENABLE_A11Y_THRESHOLD" = "true" ]; then 
+  echo "axe DevHub found $IssuesOverA11yThreshold accessibility violations over your a11y threshold."
+fi
+
+echo "See the full report on axe DevHub: $SERVER_URL$AxeURL"
```

Comment:
This is already echoed earlier on L36. I do think this prefix is more meaningful for a user than "Axe URL", but we shouldn't print it twice